spec/grape/validations/params_scope_spec.rb

```diff
@@ -466,6 +466,20 @@ def initialize(value)
       end.to raise_error(Grape::Exceptions::UnknownParameter)
     end
 
+    it 'DOES NOT raise an error if the dependent parameter is a Hash' do
```

Comment:
Lets not shout. So "does not raise error if ..." Thanks :)

Reply:
No problem.

The idea was not to shout, but to make it more visible when running using "documentation" format.